src/components/Steps/index.js

```diff
@@ -19,7 +19,7 @@ export default class Steps extends Component {
     initialStep: PropTypes.number.isRequired,
     steps: PropTypes.arrayOf(
       PropTypes.shape({
-        element: PropTypes.oneOfType([PropTypes.string, PropTypes.instanceOf(Element)]),
+        element: PropTypes.oneOfType([PropTypes.string, PropTypes.element]),
```

Comment:
I have not used `prop-types` in a very long time but I think, at least according to the documentation, that this modification may actually change the intended behavior here as `PropTypes.element` is documented as `A React element (ie. <MyComponent />)` which differs from `PropTypes.instanceOf(Element)`.

I'm not sure what should even be the proper fix for this case, maybe something like `(typeof Element === 'undefined' ? PropTypes.any : PropTypes.instanceOf(Element)`?

Any thoughts?

Reply:
After a bit of testing, this seems to work so I'll go ahead with the solution I suggested. 

I'll also add another SSR related change in another PR.